.pipelines/templates/publish-helm-chart.yaml

```diff
@@ -5,11 +5,25 @@ steps:
     displayName: Login to acr
     env: 
       REGISTRY_NAME: $(REGISTRY_NAME)
+  - script: |
+      cp $(image.dir)/$(chart.name)-*.tgz .
+      FILENAME=$(ls $(chart.name)-*.tgz)
+      TRIM=${FILENAME##$(chart.name)-}
+      echo "##vso[task.setvariable variable=CHART_TAG]${TRIM%%.tgz}"
+      echo ${CHART_TAG}
+    displayName: Get osm-arc chart version
+  - script: |
+      echo "##vso[task.setvariable variable=UNIQUE_TAG]$(git rev-parse --short HEAD)"
+      echo ${UNIQUE_TAG}
+    displayName: Create Unique Tag
+  - script: | 
+      echo "##vso[task.setvariable variable=CHECKOUT_TAG]${CHART_TAG}-${UNIQUE_TAG}-pr"
+      echo ${CHECKOUT_TAG}
+    displayName: Create Checkout Tag 
```

Comment:
Add more details to displayName

Reply:
Will rename checkout_tag to a better variable name in a follow up PR and provide more info in displayName.